src/pages/notification/index.tsx

```diff
@@ -1,3 +1,100 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { Post } from '~/api/types/postTypes';
+import ArrowBackButton from '~/common/components/ArrowBackButton';
+import Group from '~/common/components/Group';
+import Loading from '~/common/components/Loading';
+import Text from '~/common/components/Text';
+import useNotificationList from '~/common/hooks/queries/useNotificationList';
+import useLayout from '~/common/hooks/useLayout';
+import { OWHAT_TOKEN } from '~/constants/token';
+import { BrowserStorage } from '~/utils/storage';
+import { elapsedTime, isOverTwoWeeks } from '~/utils/time';
+
+import Alarm from './components/Alarm';
+import SeenButton from './components/SeenButton';
+
 export default function NotificationPage() {
-  return <section>Notification</section>;
+  const { changeMeta, changeBottomNavigator } = useLayout();
+
+  const myStorage = new BrowserStorage<string>(OWHAT_TOKEN);
+  const token = myStorage.get();
+
+  const navigate = useNavigate();
+
+  const { notificationList, isLoading } = useNotificationList();
+
+  const recentNotificationList = notificationList?.filter(
+    ({ createdAt }) => !isOverTwoWeeks(createdAt),
+  );
+
+  useEffect(() => {
+    changeBottomNavigator(true);
+    changeMeta({
+      title: '알림',
+      left: <ArrowBackButton />,
+      right: <></>,
+    });
+  }, []);
+
+  useEffect(() => {
+    if (!token) {
+      navigate('/login');
+    }
+  }, [token]);
+
+  return (
+    <section className="px-small">
+      {isLoading ? (
+        <Group spacing="sm" align="center" position="center" className="h-60">
+          <Loading />
+        </Group>
+      ) : (
+        <>
+          <Group spacing={0} position="right" className="mb-small">
+            <SeenButton />
+          </Group>
+          <ul className="scroll-none align-center flex h-full flex-col gap-3 overflow-y-auto">
+            {recentNotificationList && recentNotificationList.length > 0 ? (
+              recentNotificationList.map(
+                ({
+                  _id,
+                  seen,
+                  author,
+                  follow,
+                  comment,
+                  message,
+                  like,
+                  createdAt,
+                }) => (
+                  <Alarm
+                    key={_id}
+                    seen={seen}
+                    username={author.fullName}
+                    follow={!!follow}
+                    comment={
+                      comment
+                        ? JSON.parse((comment.post as Post).title).title
+                        : ''
+                    }
+                    message={!!message}
+                    like={
+                      like ? JSON.parse((like.post as Post).title).title : ''
+                    }
+                    date={elapsedTime(createdAt)}
+                  />
```

Comment:
이 경우에는 다음과 같이 사용할 수도 있겠네요.
```suggestion
{
  recentNotificationList.map((alarm) => 
    <Alarm key={alarm._id} {...alarm} />
}
```

작성하면서 더 자세히 보게 되어 발견했는데, 지금처럼 props로 넘겨줄 때 모든 값을 처리해서 넘겨주는 건 컴포넌트가 지나치게 명령적으로 구현될 가능성이 있는 것 같아요. 컴포넌트에 사용될 값을 어떻게 파싱해야 하는지까지 명시적으로 작성해서 넘겨주는 것보다 '이런 값이 들어가겠지' 하고 넘겨주고, 이에 대한 세부적인 처리는 컴포넌트 내에서 하는 것이 어떨까요?